PubHub/PubHub.AdminPortal/Program.cs

```diff
@@ -1,11 +1,19 @@
 using PubHub.AdminPortal.Components;
+using PubHub.Common.ApiService;
+using PubHub.Common.Extensions;
 
 var builder = WebApplication.CreateBuilder(args);
 
 // Add services to the container.
 builder.Services.AddRazorComponents()
     .AddInteractiveServerComponents();
 
+builder.Services.AddPubHubServices(new ApiOptions
+{
+    Address = builder.Configuration.GetSection("ApiEndpoint").ToString()!,
+    HttpClientName = "PubHubApi"
```

Comment:
Could we extract this into a constant (i.g `public const string PUBHUB_API`)

Reply:
You're thinking that we make a Constant with the Address and HttpClientName and then just reference that in the parameter when adding the PubHubServices, right?

Reply:
No. The Address is fine to have in `appsettings.json`, however, the magic strings are bad practise to have in code.
`ApiEndpoint` could be a constant as well as `PubHubApi`